The libxml flags must be included and DATADIR must be defined. These
are missing when building midori using autotools instead of waf.

--- midori-0.1.0/midori/Makefile.am	2008-10-19 00:46:44.000000000 +0200
+++ midori-0.1.0/midori/Makefile.am.new	2008-11-02 00:56:57.000000000 +0100
@@ -6,9 +6,10 @@
     $(GTK_CFLAGS)           \
     $(GTKSOURCEVIEW_CFLAGS) \
     $(WEBKIT_CFLAGS)        \
+    $(LIBXML_CFLAGS)        \
     -I../katze
 
-AM_CFLAGS = -DMIDORI_LOCALEDIR=\""$(localedir)"\"
+AM_CFLAGS = -DDATADIR=\""$(datadir)"\" -DMIDORI_LOCALEDIR=\""$(localedir)"\"
 
 LDADD = \
     $(UNIQUE_LIBS)       \
@@ -16,6 +17,7 @@
     $(GTK_LIBS)          \
     $(GTKSOURCEVIEW_LIBS)\
     $(WEBKIT_LIBS)       \
+    $(LIBXML_LIBS)       \
     $(INTLLIBS)          \
     ../katze/libkatze.la
 
